src/components/progress-tracker/progress-tracker.component.js

```diff
@@ -25,7 +28,9 @@ const ProgressTracker = ({
   "aria-valuetext": ariaValueText,
   size = "medium",
   length = "256px",
+  error = false,
```

Comment:
Personally I think ultimately `error` should be of type `string` only, as we would want to display an error message where "error" is in the example below, so users can clearly see what caused the error.

<img width="305" alt="Screenshot 2022-11-30 at 15 34 02" src="https://user-images.githubusercontent.com/18368713/204840559-1a2d7f13-b6fc-4ab0-b493-f5d1f3954129.png">

This would be similar to how validation styling used by components such as [Textbox](https://carbon.sage.com/?path=/docs/textbox--validations-string-new-design#new-designs-validation) and [Textarea](https://carbon.sage.com/?path=/docs/textarea--validations-string-new-design#new-designs-validation):

<img width="273" alt="Screenshot 2022-11-30 at 15 36 24" src="https://user-images.githubusercontent.com/18368713/204841108-9c5f3794-83b1-4440-9527-d3a5496ccc50.png">

<img width="276" alt="Screenshot 2022-11-30 at 15 36 01" src="https://user-images.githubusercontent.com/18368713/204841029-1e2da7ed-4b67-4c58-88b6-4d00e2e5f79e.png">

I realise you have implemented the design you have been given here, which does not clearly show how error messages should be handled. Could you feedback this to the designer please? As I am wary this would be inconsistent with Design System's latest approach to validation messages.

Reply:
So this a part of the larger discussions we had before -  how the error is going to be indicated by the progress tracker itself and how do we define error. I am not sure if tracking progress and error handling on input fields have the same idea ...

Reply:
I think it would be fine to leave it as a boolean for now but expand it to support string as well if the need arises. As there's no designs for the latter it doesn't make sense to handle for it yet imo